CHANGELOG.md

```diff
@@ -21,28 +21,32 @@
 
 ## BUGFIXES -->
 
+# dimensionality_reduction 0.2.1 2024-12-18
+
+## MAJOR CHANGES
+
+* Only calculate distances between waypoint cells instead of between all cells and waypoint cells (PR #14)
+* Only calculate distances between waypoint cells and label centroids instead of all cells and label centroids (PR #14)
+* Use between waypoint distances instead of cell-waypoint distances and waypoint-centroid distances instead of cell-centroid distances in distance correlation metrics (PR #14)
+
 # dimensionality_reduction 0.2.0 2024-12-09
 
 ## NEW FUNCTIONALITY
 
-* Add calculation of distances to/between waypoints and label centroids to dataset pre-processing
-* Add a post-processing component that calculates distances in the embedding space
-* Add to centroid and between label distance correlation scores
+* Add calculation of distances to/between waypoints and label centroids to dataset pre-processing (PR #11)
+* Add a post-processing component that calculates distances in the embedding space (PR #11)
+* Add to centroid and between label distance correlation scores (PR #11)
 
 ## MAJOR CHANGES
 
-* Modify co-ranking metrics to use pre-computed distances
-* Modify distance correlation metrics to use pre-computed distances
-* Move spectral distance correlation to a separate component
-* Disable the trustworthiness metric as it is calculated as part of the co-ranking metrics
+* Modify co-ranking metrics to use pre-computed distances (PR #11)
+* Modify distance correlation metrics to use pre-computed distances (PR #11)
+* Move spectral distance correlation to a separate component (PR #11)
+* Disable the trustworthiness metric as it is calculated as part of the co-ranking metrics (PR #11)
 
```

Comment:
```suggestion
* Only calculate distances between waypoint cells instead of between all cells and waypoint cells (PR #14)
* Only calculate distances between waypoint cells and label centroids instead of all cells and label centroids (PR #14)
* Use between waypoint distances instead of cell-waypoint distances and waypoint-centroid distances instead of cell-centroid distances in distance correlation metrics (PR #14)

```